### ISSUE TITLE
Tree View does not follow Look and Feel Font size

### ISSUE THREAD

**Issue by johnblommers** [vkbo/novelWriter#273]
I grabbed the version 0.7 branch ahead of schedule and noticed that the Tree View pane's font size does not follow the Preferences&rarr;General&rarr;Look and Feel&rarr;font size&rarr;value.

I would have expected the font to follow this setting but I may be missing something. The check marks **do** appear to follow the font size. Screen shots attached illustrate:

**Big font size:**
![Big](https://user-images.githubusercontent.com/8484427/83362762-8b675800-a348-11ea-9ed9-6d0b5b1a1bc3.png)

**Little font size:**
![Small](https://user-images.githubusercontent.com/8484427/83362766-93bf9300-a348-11ea-8798-284139915504.png)

**Comment by vkbo** [comment]
Yeah, I've noticed across various platforms that some elements can be controlled, others cannot. On Windows 10 the menu doesn't scale for instance.

The tree scales just fine on Debian sid. so this is clearly another such exception. There is no hard coded sizes for the tree view.

**Comment by vkbo** [comment]
I also see that the document title/path follows the GUI font size. It should actually follow the document.

**Comment by johnblommers** [comment]
I should have added that my observations above were made on Ubuntu 19.10

**Comment by vkbo** [comment]
Yeah, I recognise the Ubuntu icons :)

I have virtual machines of the previous 4 LTS releases, so I'll see if I can force the tree to follow the sizes.

Which version of Qt are you using? Should be around the top of the log output if you run novelWriter with `--info`, `--debug`, or `--verbose`.

**Comment by vkbo** [comment]
Confirmed on Ubuntu 18.04.

Explicitly setting the font per line in the tree seems to fix it, so I'll commit that fix.
![image](https://user-images.githubusercontent.com/1619840/83396368-29682a80-a3fc-11ea-9f29-a83d32010747.png)

**Comment by johnblommers** [comment]
```
[2020-06-01 11:36:07]                     nw.guimain:64    INFO      Kernel: 5.3.0-7625-generic
[2020-06-01 11:36:07]                     nw.guimain:65    INFO      Host: blacknight
[2020-06-01 11:36:07]                     nw.guimain:67    INFO      Qt5 Version: 5.12.4 (51204)
[2020-06-01 11:36:07]                     nw.guimain:70    INFO      PyQt5 Version: 5.12.3 (51203)
[2020-06-01 11:36:07]                     nw.guimain:73    INFO      Python Version: 3.7.5rc1 (0x30705c1)
```

**Comment by vkbo** [comment]
I merged the fix before releasing 0.7, so it should work now. I fixed 4 of the 5 widgets where this was an issue. The only place it doesn't scale is the session log dialog. I will deal with that later. I will probably redesign it anyway.

**Comment by vkbo** [comment]
Just for the record, and for later reference, the issue here was actually caused by a CSS file entry from early in development, and entirely forgotten. The CSS entry fixed the font size of the QTreeWidget to 13px. That is why the internal setting didn't override it. Interestingly, it seems to vary between platforms which of the two contradicting settings took precedence.
